### PR TITLE
Followup: fix(settings): Don't display built-in task types in AI settings TextProcessing

### PR DESCRIPTION
**Pull request by marcelklehr**
## Checklist

- Code is [properly formatted](https://docs.nextcloud.com/server/latest/developer_manual/digging_deeper/continuous_integration.html#linting)
- [Sign-off message](https://github.com/src-d/guide/blob/master/developer-community/fix-DCO.md) is added to all commits
- [ ] Tests ([unit](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#unit-tests), [integration](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#integration-tests), api and/or acceptance) are included
- [ ] Screenshots before/after for front-end changes
- [ ] Documentation ([manuals](https://github.com/nextcloud/documentation/) or wiki) has been updated or is not required
- [ ] [Backports requested](https://github.com/nextcloud/backportbot/#usage) where applicable (ex: critical bugfixes)

### CODE REVIEW

File: apps/settings/src/components/AdminAI.vue [hunk 1]
```diff
@@ -90,9 +90,9 @@
 					<p>&nbsp;</p>
 				</div>
 			</template>
-			<template v-if="!hasTextProcessing">
+			<template v-if="tpTaskTypes.length === 0">
 				<NcNoteCard type="info">
-					{{ t('settings', 'None of your currently installed apps provide Text processing functionality') }}
+					{{ t('settings', 'None of your currently installed apps provide custom Text processing functionality.') }}
```
[the remark this answers is not in the document]
Comment (marcelklehr) [reply apps/settings/src/components/AdminAI.vue:95]: It's the name of an API

[the remark this answers is not in the document]
Comment (marcelklehr) [reply apps/settings/src/components/AdminAI.vue:95]:
```suggestion
					<!-- TRANSLATORS Text processing is the name of a Nextcloud-internal API -->
					{{ t('settings', 'None of your currently installed apps provide custom Text processing functionality.') }}
```

**Review by miaulalala — APPROVED** [review]

**Review by susnux — APPROVED** [review]

**Pull request merged**